src/index.html

```diff
@@ -3,11 +3,294 @@
+              src="./images/bike-description-image-1.jpg"
+              class="pictures__image"
+              alt="Auto-Unlock"
+              >
+            </div>
+
+            <div class="pictures__item pictures__item--small">
+              <img
+              src="./images/bike-description-image-2.jpg"
+              class="pictures__image"
+              alt="Auto-Unlock"
+              >
+            </div>
+          </div>
+          <h4 class="title article__title">
+            Auto Unlock
+          </h4>
+          <p class="article__paragraph">
+            The app senses when you're nearby to unlock automatically. GPS tracking so you know where your bike is and can track it anytime.
+          </p>
+        </article>
+
+        <article class="article section__article">
+          <div class="pictures article__pictures">
+            <div class="pictures__item pictures__item--big">
+              <img
+              src="./images/bike-description-image-3.jpg"
+              class="pictures__image"
+              alt="Range & Integrated lights"
+              >
+            </div>
+
+            <div class="pictures__item pictures__item--small">
+              <img
+              src="./images/bike-description-image-4.jpg"
+              class="pictures__image"
+              alt="Range & Integrated lights"
+              >
+            </div>
+          </div>
+          <h4 class="title article__title">
+            Range & Integrated lights
+          </h4>
+          <p class="article__paragraph">
+            The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+          </p>
+        </article>
+
+        <article class="article section__article">
+          <div class="pictures article__pictures">
+            <div class="pictures__item pictures__item--small">
+              <img
+              src="./images/bike-description-image-5.jpg"
+              class="pictures__image"
+              alt="Hydraulic disc brakes & Lightweight"
+              >
+            </div>
+
+            <div class="pictures__item pictures__item--big">
+              <img
+              src="./images/bike-description-image-6.jpg"
+              class="pictures__image"
+              alt="Hydraulic disc brakes & Lightweight"
+              >
+            </div>
+          </div>
+          <h4 class="title article__title">
+            Hydraulic disc brakes & Lightweight
+          </h4>
+          <p class="article__paragraph">
+            Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+          </p>
+        </article>
+
+        <a href="#" class="button section__button">Explore</a>
+      </section>
+
+      <section class="section main__section" id="contact-us">
```

Comment:
Semantically you shouldn't wrap your form in a section "<section> is a generic sectioning element, and should only be used if there isn't a more specific element to represent it" -> [Source](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)